src/cryptoadvance/specter/server_endpoints/wallets_api.py

```diff
@@ -489,15 +491,35 @@ def addressinfo(wallet_alias):
         wallet = app.specter.wallet_manager.get_by_alias(wallet_alias)
         address = request.form.get("address", "")
         if address:
-            descriptor = wallet.get_descriptor(address=address)
+            descriptor = add_checksum(
+                wallet.get_descriptor(address=address, keep_xpubs=False).to_string()
+            )
+            xpubs_descriptor = add_checksum(
+                wallet.get_descriptor(address=address, keep_xpubs=True).to_string()
+            )
+            # The last two regex groups are optional since Electrum's derivation path is shorter
+            derivation_path_pattern = (
+                r"(\/[0-9h]+)(\/[0-9h]+)(\/[0-9h]+)(\/[0-9h]+)?(\/[0-9h]+)?"
+            )
+            # Only "descriptor" gives full derivation path, looks usually like this:
+            # wpkh([8c24a510/84h/1h/0h/0/0]0331edcb16cfd ... e02552539d984)#35zjhlhm
+            match = re.search(derivation_path_pattern, descriptor)
+            if not match:
+                logger.debug(
+                    "Derivation path could not be parsed. Sth. wrong with the regex?"
```

Comment:
Always pass as much data into a debug/error message as possbile. This message is not helpful if someone is reading it. At best, pass in descriptor AND pattern.

Reply:
Changed as requested in https://github.com/cryptoadvance/specter-desktop/pull/1567/commits/94f6bef238d516b886a3e8bd8ef0fd5a34b41c18